os/drivers/audio/alc5658.c

```diff
@@ -1170,6 +1287,10 @@ static int alc5658_interrupt(FAR const struct alc5658_lower_s *lower, FAR void *
 
 	DEBUGASSERT(lower && priv);
 
+	if (!priv || !lower) {
+		return ERROR;
```

Comment:
if priv or lower is not exist, It means there's no device.
Hence ENXIO is correct instead of ERROR.
How do you think?

Reply:
Initially I thought the same. But if you look at other APis, they use OK and ERROR. So I also followed the same to keep it uniform across file.

Reply:
Let this be the first step for others too. It is pending for long time.